config/config.go

```diff
@@ -160,6 +165,49 @@ func ExtractDependencies(descriptor *FlogoAppDescriptor) []*Dependency {
 	return dh.deps
 }
 
+func ExtractAllDependencies(appJson string) ([]*Dependency) {
+
+	dh := &depHolder{}
+
+	var descriptor interface{}
+
+	//Should be valid app json
+	json.Unmarshal([]byte(appJson), &descriptor)
```

Comment:
can we catch any error here?

Reply:
@mellistibco: The App json is already parsed in earlier step. Any json related issues will be caught there.